doc: Add id and eui errors to gw troubleshooting (#525)
TheThingsIndustries/lorawan-stack-docs: doc/content/gateways/troubleshooting/_index.md

- Checking your internet connection
- Using {{% tts %}} cluster closest to the location of your gateway

## I Get a "ID Already Taken" Error When Adding a Gateway.

Another gateway is already registered with this ID. This gateway may have been deleted already, but {{% tts %}} reserves previously used IDs for [security reasons]({{< ref "reference/id-eui-constraints" >}}).

> This gateway may also be registered by another user, and if you are not an administrator (e.g if you are using The Things Stack Community Edition) you will not be able to see gateways registered by other users.

Reviewer: ```suggestion
This gateway may also be registered by another user, but you are not able to see gateways registered by other users if you are not an administrator (e.g. if you are using {{% tts %}} Community Edition).
```

To solve this, use a different gateway ID. If you are an administrator and wish to reuse a deleted ID, see [Purging Entities]({{< ref "reference/purge" >}}).

## I Get a "Gateway with EUI is Already Registered" Error When Adding a Gateway.

> Another gateway is already registered with the same Gateway EUI. This gateway may be registered by another user, and if you are not an administrator (e.g if you are using The Things Stack Community Edition) you will not be able to see gateways registered by other users.

Reviewer: ```suggestion
Another gateway is already registered with the same Gateway EUI. This gateway may be registered by another user, but if you are not an administrator (e.g if you are using The Things Stack Community Edition) you will not be able to see gateways registered by other users.
```
